### DISCUSSION TITLE
Github connection erros.

### DISCUSSION THREAD

**Discussion by y-yudai** [vercel/vercel#5545] [Help]
Hi! I've recently moved my repos to a GitHub organization. I understand Vercel requires a team in order to connect to GH orgs, so I've created a team and am on the 14 day trial. And, Installed Vercel on GH orgs already. But, Github connection erros. What should I confirm?

**Comment by markozxuu** [comment]
Hi @y-yudai  ! Could you give me more information about your error, such as a log or something similar

**Reply by y-yudai** [reply]
Hi @markozxuu 
Thank you for the comment.
I get an error with the attached image when importing.
<img width="768" alt="ss_error_message" src="https://user-images.githubusercontent.com/20409002/101882737-9f362580-3bd9-11eb-9b7a-79e076e8aa45.png">

I also checked the Github permissions and they seem to be fine.
<img width="838" alt="ss_vercel_permissions" src="https://user-images.githubusercontent.com/20409002/101882748-a1987f80-3bd9-11eb-88cf-96c0a7934498.png">
